spec/system/admin/instrument_price_policies_controller_spec.rb

```diff
@@ -154,6 +154,12 @@
         expect(page).to have_no_link "Remove"
       end
     end
+
+    describe "with hidden price policies", :js do
```

Comment:
Do we need `:js` here?

Reply:
Well, I can't tell the exact reason behind this, but if I remove the :js flag the tests fail. Capybara isn't able to find some fields